pages/impressum.vue

```diff
@@ -0,0 +1,157 @@
+
+            <div class="content">
+                <h2>Contributors</h2>
+                <ul class="lists">
+                    <li>Anna Blasinger</li>
+                    <li>Florent Gabillon</li>
+                    <li>Alica Holla</li>
+                    <li>Vanessa Köck</li>
+                    <li>Juliana Matsumura</li>
+                    <li>Antoine Muneret</li>
+                    <li>Hannah Wagner</li>
+                </ul>
+            </div>
+
+            <div class="content">
+                <h2>Supervision</h2>
+                <p>Dr. Victor-Adriel De-Jesus-Oliveira, MSc</p>
+            </div>
+            <div class="content">
+                <h2>Media owner</h2>
+
+                <ul class="lists">
+                    <li class="font-bold">Fachhochschule St. Pölten GmbH</li>
+                    <li>Campus-Platz 1</li>
+                    <li>3100 St. Pölten</li>
+                    <li>Austria</li>
+                </ul>
+
+                <p>
+                    <NuxtLink
+                        class="linkText"
+                        to="https://www.fhstp.ac.at/de/impressum/impressum?set_language=de"
+                        target="_blank"
+                        >Imprint </NuxtLink
+                    >,
+                    <NuxtLink
+                        class="linkText"
+                        to="https://www.fhstp.ac.at/en/legal-information/legal-information?set_language=en"
+                        target="_blank"
+                        >Legal Information</NuxtLink
+                    >
+                </p>
+            </div>
+
+            <div class="content">
+                <h2>Licence</h2>
+                <p>
+                    comixcraft and the used
+                    <NuxtLink class="linkText" to="https://fhstp.github.io/comixplain/" target="_blank"
+                        >Comixplain
+                    </NuxtLink>
+                    assets are CC BY-SA licensed, and you can create comics and share them under the same license.
+                </p>
+            </div>
+
+            <div class="content">
+                <h2>GitHub</h2>
+                <p>
+                    Comixcraft is an open-source project and the repository is accessible on
+                    <NuxtLink class="linkText" to="https://github.com/comixcraft" target="_blank">GitHub</NuxtLink>.
+                </p>
+            </div>
+        </div>
+    </div>
+
+    <FooterComponent />
+</template>
+
+<style scoped lang="scss">
+    .impressum {
+        display: flex;
+        flex-direction: column;
+        gap: $spacer-4;
+        padding: $spacer-5;
+    }
+
+    .content {
+        display: flex;
+        flex-direction: column;
+        gap: $spacer-3;
+    }
+
+    .top-nav {
+        justify-content: flex-start;
+    }
+
+    .top-nav__logo {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        width: 10rem;
+    }
+
+    .linkText {
```

Comment:
this class should be named link-text

Reply:
not link_text as the other comments say? 
I changed it to that, link_text but please let me know :)